### DISCUSSION TITLE
'PyInstaller.depend.bindepend' has no attribute 'selectImports'

### DISCUSSION THREAD

**Discussion by Patricklevyy** [pyinstaller/pyinstaller#8057] [Help]
Hello, 
I'm trying to convert my .py file to .exe with py installer. I have a simple code that uses kivy. But I have an error that I can't fix. When I launch pyinstaller I get after a few moments:
`AttributeError: module 'PyInstaller.depend.bindepend' has no attribute 'selectImports'`

**Comment by rokm** [comment]
`PyInstaller.depend.bindepend.selectImports` was refactored into low-level `PyInstaller.depend.bindepend.get_imports` and high-level `PyInstaller.depend.bindepend.binary_dependency_analysis` in v.6.0 (#7784). 

So if `kivy` hook is using `selectImports`, it should be fixed accordingly. As the hook in question is maintained as part of `kivy`, you should report the issue to them. (As a side note, I am not sure why `kivy` hook would need to perform binary dependency analysis itself - it should just return the top-level binaries and let PyInstaller handle the dependencies).
